.env.production

```diff
@@ -0,0 +1,2 @@
+VUE_APP_OIDC_AUTHORITY=https://auth.sso.interieur.rie.gouv.fr/.well-known/openid-configuration
+VUE_APP_OIDC_CLIENT_ID=basegun
```

Comment:
same remark as .env.dev
How do you transmit these variables at deployment time ?

Reply:
When using `npm run dev` it will be automatically used